bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -21,6 +21,21 @@
 				<type>tesla:trafficminutesdelay</type>
 			</add-channel>
 		</instruction-set>
+
+		<instruction-set targetVersion="2">
+			<add-channel id="softwareupdateavailable" typeId="softwareupdateavailable">
```

Comment:
For all update instructions, i miss the drop channel for car version (depends on other comments)